scripts/process_vulnerabilities.py

```
@@ -194,26 +211,108 @@ def _try_assign_to_fallback(self, issue_number: int) -> None:
+            body += f"""### Steps for @copilot
 
-        body = f"""## Security Vulnerability Detected
+1. Update the `{package}` dependency to `{recommended_version}` or another secure version
+2. Review the vulnerability details by clicking on each vulnerability ID above
+3. Update any related dependencies if needed
+4. Run tests to ensure compatibility
+5. Create a pull request with the security fix
+
+---
+
+**ℹ️ Note**: If GitHub Copilot is enabled in your repository, you can assign this issue to the Copilot coding agent for automated remediation. Simply assign this issue to `@copilot` or your configured Copilot agent username.
```

Ruff (0.14.13)

String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)